handel/src/services/aurora/index.ts

```diff
@@ -202,7 +203,7 @@ export class Service implements ServiceDeployer {
         const stack = await awsCalls.cloudFormation.getStack(stackName);
         if (!stack) {
             const dbUsername = rdsDeployersCommon.getNewDbUsername();
-            const dbPassword = rdsDeployersCommon.getNewDbPassword();
```

Comment:
Let's keep using the rdsDeployersCommon instead of using the crypto random string directly. But we should modify the getNewDbUsername and getNewDbPassword functions in the RDS deployers common module to use the crypto random string generation.

Likewise let's make this change on the Aurora Serverless, MySQL, and PostgreSQL deployers.

Reply:
When I looked into the code, I realized that the rdsDeployersCommon.getNewDbPassword already calls on the same crypto random string generator

Reply:
Oh cool, so really I guess we only need to change Amazon MQ then.